icu4c/source/data/BUILDRULES.py

```diff
@@ -308,6 +310,36 @@ def generate_coll_ucadata(config, glob, common_vars):
     ]
 
 
+def generate_full_unicore_data(config, glob, common_vars):
+    # The core Unicode properties files (pnames.icu, uprops.icu, ucase.icu, ubidi.icu)
+    # are hardcoded in the common DLL and therefore not included in the data package any more.
+    # They are not built by default but need to be built for ICU4J data and for getting the .c source files
+    # when updating the Unicode data.
```

Comment:
I believe that the part "and for getting the .c source files when updating the Unicode data" is no longer true.
Please remove.

Instead we can say something like we need these for ICU4J, both in the .jar and in the .dat file (if ICU4J uses the .dat file).

Reply:
Done